<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>temp conversion:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>Celsius</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Farenheit</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Kelvin</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>to</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QComboBox" name="comboBox_2">
           <item>
            <property name="text">
             <string>Celsius</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Farenheit</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Kelvin</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>length conversion:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QComboBox" name="comboBox_3">
           <item>
            <property name="text">
             <string>mm</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>dm</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>cm</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Meter</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>km</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>mile</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>foot</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>inch</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>to</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QComboBox" name="comboBox_4">
           <item>
            <property name="text">
             <string>mm</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>dm</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>cm</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Meter</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>km</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>mile</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>foot</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>inch</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_4"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>velocity conversion:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QComboBox" name="comboBox_5">
           <item>
            <property name="text">
             <string>km/h</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>mile/h</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>m/s</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>mach</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_5"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>to</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QComboBox" name="comboBox_6">
           <item>
            <property name="text">
             <string>km/h</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>mile/h</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>m/s</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>mach</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_6"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
